typescript/cognito-idp/copy-dist.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copy files to the dist folder after a build.
+ */
+
+const fs = require('fs-extra');
+
+fs.ensureDirSync('dist');
+fs.ensureDirSync('dist/web');
+fs.ensureDirSync('dist/web/img');
+fs.ensureDirSync('dist/lambda');
+fs.ensureDirSync('dist/lambda/node_modules');
+fs.ensureDirSync('functions/node_modules');
+
+fs.emptyDirSync('dist');
+
```

Comment:
I would recommend moving all build/dev related scripts/tools to a “tools” or “scripts” directory so they won’t mix up with the actual app.